code/modules/mob/dead/dead.dm

```diff
@@ -108,11 +108,15 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 			else
 				PL2 += "[PL[i]], "
 
+		var/str_job = job.title
+		if(job.title == "Court Agent")
+			str_job = "Adventurer"
```

Comment:
this will cause two instances of adventurers to appear in the list

Reply:
Guh, didn't notice that.

I guess at this point it'd be simpler to just not have them show up in the lobby at all.

Reply:
probably a better choice, its not like anyone's going to pay attention to 20+ names, _plus early latejoins exist at the roundstart mark to blend in as plasuable deniable meta anyway._